doc/plotting.rst

```diff
@@ -45,16 +48,18 @@ Imports
     # Use defaults so we don't get gridlines in generated docs
     import matplotlib as mpl
     mpl.rcdefaults()
+    mpl.rcParams.update({'figure.autolayout': True})
```

Comment:
Maybe this is undoing the seaborn import? If it doesn't seem to be changing the doc build, take it out.